GVRf/Framework/jni/engine/memory/gl_delete.h

```diff
@@ -25,15 +28,48 @@
 #define GVR_INVALID 0
 
 namespace gvr {
+
+extern pthread_key_t deleter_key;
```

Comment:
Add a few things that caught my eyes. This is internal data. Suggest removing from gl_delete.h.